common/src/main/java/com/alibaba/nacos/common/utils/MD5Utils.java

```diff
@@ -26,10 +26,10 @@
  */
 @SuppressWarnings("PMD.ClassNamingShouldBeCamelRule")
 public class MD5Utils {
-    
+
```

Comment:
Please do not change the indent. Please use nacos' code style to reformat codes.

Reply:
ok, i will modify it.